eclipse-formatter.xml

```diff
@@ -9,25 +9,25 @@
         <setting id="org.eclipse.jdt.core.compiler.source" value="10" />
         <setting id="org.eclipse.jdt.core.formatter.align_fields_grouping_blank_lines" value="2147483647" />
         <setting id="org.eclipse.jdt.core.formatter.align_type_members_on_columns" value="false" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_allocation_expression" value="16" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_annotation" value="0" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_enum_constant" value="16" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_explicit_constructor_call" value="16" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_method_invocation" value="16" />
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_qualified_allocation_expression" value="16" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_allocation_expression" value="48" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_annotation" value="48" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_enum_constant" value="48" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_explicit_constructor_call" value="48" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_method_invocation" value="48" />
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_qualified_allocation_expression" value="48" />
```

Comment:
@obj-obj  - these "magic" values are bitflags defined here: https://www.cct.lsu.edu/~rguidry/ecl31docs/api/constant-values.html#org.eclipse.jdt.internal.formatter.align.Alignment.M_INDENT_ON_COLUMN
See `createAlignmentValue` in https://www.cct.lsu.edu/~rguidry/ecl31docs/api/src-html/org/eclipse/jdt/core/formatter/DefaultCodeFormatterConstants.html for which constants are valid

Reply:
Ayyy actual documentation?